com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md

```diff
@@ -44,8 +44,14 @@ After you install the NVIDIA package, more properties appear in [HDRP Assets](HD
 Your Unity project now supports DLSS and you can now enable DLSS for Cameras in your scene. Enabling DLSS in the HDRP Asset exposes other properties that you can use to customize DLSS. For information about these properties, see the [HDRP Asset](HDRP-Asset.md) documentation.
 
 1. In the Hierarchy or Scene view, select a Camera and view it in the Inspector.
-2. Select **Allow DLSS**.
-3. Enabling **Allow DLSS** exposes other properties that you can use to customize DLSS for the Camera. For information about these properties, see the [Camera](HDRP-Camera.md) documentation.
+2. Select **Allow Dynamic Resolution** to expose the DLSS settings. For more information see the [Dynamic Resolution](Dynamic-Resolution.md) guide. 
+3. Select **Allow DLSS**.
+4. Enabling **Allow DLSS** exposes other properties that you can use to customize DLSS for the Camera. For information about these properties, see the [Camera](HDRP-Camera.md) documentation.
```

Comment:
Remove passive voice: 

Enable **Allow DLSS** to expose other properties that you can use to customize DLSS for the Camera.

Reply:
Done! thank you.